.github/workflows/release.yml

```
@@ -100,30 +100,30 @@ jobs:
           release_version=${GITHUB_REF:11}
           sed -i "s/^projectVersion.*$/projectVersion\=${release_version}/" gradle.properties
           echo "release_version=${release_version}" >> $GITHUB_OUTPUT
-      - name: Setup GraalVM CE
+      - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1.0.12
```

```suggestion
        uses: graalvm/setup-graalvm@v1.1.4.2
```

[v1.1.4.2](https://github.com/graalvm/setup-graalvm/releases/tag/v1.1.4.2)